src/test/java/testresults/PostTestResultsNegMainLvlSubmitted.java

```diff
@@ -961,6 +960,5 @@ public void testResultsTestTypesMissing() {
         testResultsSteps.statusCodeShouldBe(400);
         testResultsSteps.validatePostErrorData("testTypes", "is required");
     }
-
```

Comment:
Can these formatting changes be reverted

Reply:
no need for an extra blank line there